webapp/src/main/webapp/src/locale/messages.pt.xlf

```diff
@@ -296,6 +296,12 @@
 				<target>Eventos Pendentes</target>
 			</segment>
 		</unit>
+		<unit id="receiver-pending-events">
+			<segment>
+				<source>Pending Events</source>
```

Comment:
They are **Sync Items** and not really events, we use events only on the sender side in some debezium events queue table

Reply:
Fixed. Thanks.